xbmc/addons/AddonManager.cpp

```diff
@@ -564,6 +567,40 @@ bool CAddonMgr::FindInstallableById(const std::string& addonId, AddonPtr& result
   return true;
 }
 
+bool CAddonMgr::GetInstalledBinaryAddons(BINARY_ADDON_LIST& binaryAddonList)
```

Comment:
I'm sure it can. Try harder. Mostly it does not work, because functions caked from this method are not convert, but they could and should be. See for example my other comment re GetPlatformLibraryName which currently is not const, but could even be static.